source/tests/includes/PluginTestCase.php

```diff
@@ -23,8 +23,8 @@ public function setUp(): void
 
         Functions\stubTranslationFunctions();
 
-        if (!defined('AVM_TEXT_DOMAIN')) {
-            define('AVM_TEXT_DOMAIN', 'avm-text-domain');
+        if (!defined(''api-volunteer-manager'')) {
+            define(''api-volunteer-manager'', 'avm-text-domain');
         }
```

Comment:
These lines are breaking tests due to syntas error. Remove these lines since we are not using the const for translations anymore.
```suggestion
```